### ISSUE TITLE
Unable to remove corrupted movie

### ISSUE THREAD

**Issue by Phantomx2** [seerr-team/seerr#1029]
### Description

I have a movie in my watchlist that became "not found" somehow.
There is a Clear Data button under it, but when I press it, nothing happens.

### Version

2.0.1

### Steps to Reproduce

- Not sure how it happened, but the "Movie not found" tile appears in my watchlist on the home page.
- Press Clear Data button
- Movie remains there

### Screenshots

![image](https://github.com/user-attachments/assets/7870b76c-f2bb-4b00-94c3-7784300b257c)

### Platform

desktop

### Device

PC

### Operating System

Windows 11

### Browser

Chrome, Firefox

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
It's not a corrupted media. That media has been removed from tmdb. It's just cached for now. Should be cleared when your tmdb cache gets renewed

**Comment by Phantomx2** [comment]
Is there a way to manually renew this cache? 
The missing movie has been there for months now, not getting cleared.
I tried clearing everything by the settings/Job & Caches, but didn't help.

**Comment by Fallenbagel** [comment]
> Is there a way to manually renew this cache? 
> The missing movie has been there for months now, not getting cleared.
> I tried clearing everything by the settings/Job & Caches, but didn't help.

`flush cache` tmdb didn't help? Did you try pressing clear data and then flush cache?

**Comment by Phantomx2** [comment]
> `flush cache` tmdb didn't help? Did you try pressing clear data and then flush cache?
yes, I did first press the clear data button then flush cache. 
Now in Settings / Job & Caches I can see The Movie Database (tmdb) image cached / cache size = 0, but the missing movie is still showing up.

Pressing the clear button outputs this error in the log BTW:
Could not find any entity of type "Media" matching: { "where": { "id": 1261287 } }

**Comment by Sinlocu** [comment]
Hi, I'm also having this issue. I've clicked on clear data and flushed all my caches in the web UI and the issue persists. I'm running the latest version of Jellyseer 2.0.1 on TrueNAS SCALE ElectricEEL 24.10.0.1

![image](https://github.com/user-attachments/assets/8c209314-f872-4bc3-8782-328792ce2c5b)

**Comment by Fallenbagel** [comment]
> Hi, I'm also having this issue. I've clicked on clear data and flushed all my caches in the web UI and the issue persists. I'm running the latest version of Jellyseer 2.0.1 on TrueNAS SCALE ElectricEEL 24.10.0.1
> 
> ![image](https://private-user-images.githubusercontent.com/114628408/384245911-8c209314-f872-4bc3-8782-328792ce2c5b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE5OTU2NDEsIm5iZiI6MTczMTk5NTM0MSwicGF0aCI6Ii8xMTQ2Mjg0MDgvMzg0MjQ1OTExLThjMjA5MzE0LWY4NzItNGJjMy04NzgyLTMyODc5MmNlMmM1Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExOVQwNTQ5MDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MTBkZTY5NjM4YTZmYWQ3NGU0NDM1NWJiY2MzNmZiZDFkOWJmMjg0OThjMTZhYmUwZjZiMjc5ZjIwNmNjNGVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.BFac5b14mtNoqtexwL3ir9cD_xggZ7b2CfXpD-xZFMA)

Your issue is different. Please hop onto discord to troubleshoot. This issue is about a media thats removed. Your issue seems to be entirety of tmdb not being reached

**Comment by gauthier-th** [comment]
Hi @Phantomx2, is this still an issue?

**Comment by Fallenbagel** [comment]
> > Hi [@Phantomx2](https://github.com/Phantomx2), is this still an issue?
> 
> Hi,
> I don't have any new ones, but the one I had is still there and I cannot remove it yes.
> Running 2.3.0 now
> 
> Thanks!

Hmm that's weird. I had a few of them too because some series i had were removed on tmdb. So what I did was I just pressed clear data and it cleared for me. 

(Though I'm running on pgsql)

**Comment by hbilbo** [comment]
Just to add another data point here, I had this issue as well and running clear data and flush tmdb cache appears to have fixed it for me. I am running 2.3.0 on Truenas Scale 24.10.2. Will update if it appears again.

**Comment by fallenbagel** [comment]
> Just to add another data point here, I had this issue as well and running clear data and flush tmdb cache appears to have fixed it for me. I am running 2.3.0 on Truenas Scale 24.10.2. Will update if it appears again.

The issue comes from tmdb removing a media. So the cache remains. Which is why flushing cache and clearing data is the way to fix stale media

**Comment by hbilbo** [comment]
So it appears I spoke too soon. I'm now running 2.5.0 and running "clear data" then flushing all of the cache still results in the missing media showing up eventually. I'm assuming it's because the item is still in my library but not showing up on tmdb? Not sure why it shows back up after clearing the data and cache. I am also not sure what it is because the logs just show the tmdb ID and that it can't find it.

**Comment by fallenbagel** [comment]
> So it appears I spoke too soon. I'm now running 2.5.0 and running "clear data" then flushing all of the cache still results in the missing media showing up eventually. I'm assuming it's because the item is still in my library but not showing up on tmdb? Not sure why it shows back up after clearing the data and cache. I am also not sure what it is because the logs just show the tmdb ID and that it can't find it.

Yeah you have an item in your library that has that tmdbid. So what happens is when you clear it and a sync happens it comes back but there's no tmdbid to match

**Comment by Phantomx2** [comment]
The clear data button doesn't do anything on my side, is there a way to manually remove that entry?

**Comment by fallenbagel** [comment]
> The clear data button doesn't do anything on my side, is there a way to manually remove that entry?

Through db editing yes you can. Are you on sqlite or postgresql?

I would recommend if possible hopping onto discord if you want instructions on how to traverse through the db as it's our main support channel and it's easier to provide support than in gh issues c:

**Comment by Phantomx2** [comment]
So we manually deleted the entry in the database to fix the issue.
